### ISSUE TITLE
Install Failed

### ISSUE THREAD

**Issue by Codematic-AppDev** [kyleneideck/BackgroundMusic#29]
Cloned/downloaded and ran the intall script from terminal. Install failed and the following is what I got from Terminal:

_Installing BGMXPCHelper.xpc to /Library/Application Support/Background Music.
Installing Background Music.app to /Applications.
Restarting coreaudiod to load BGMDriver.
Invalid signal.
Usage: launchctl kill <signal-number|signal-name> <service-target>
ERROR: Install script failed at line 200. This is probably a bug in the script. Feel free to report it._

My setup: MacBook Pro (Retina, 15-inch, Mid 2015) running OSX 10.10.5 (Yosemite)

**Comment by ariera** [comment]
I have a different error. Not sure if related/helpful

```
bash-3.2$ ./build_and_install.sh
Installing the virtual audio device Background Music Device.driver to /Library/Audio/Plug-Ins/HAL.
Installing BGMXPCHelper.xpc to /Library/Application Support/Background Music.
.ERROR: Install script failed at line 170. This is probably a bug in the script. Feel free to report it.
ERROR: Install script failed at line 60. This is probably a bug in the script. Feel free to report it.
ERROR: Build step failed. See build_and_install.log for details.
Failed command:
    sudo xcodebuild -project BGMApp/BGMApp.xcodeproj -target BGMXPCHelper -configuration Release RUN_CLANG_STATIC_ANALYZER=0 DSTROOT=/ INSTALL_PATH=/Library/Application Support/Background Music install
```

**Comment by kyleneideck** [comment]
@Codematic-AppDev Thanks for reporting this. It should be fixed in 0178644417ead43b4df83fb331cb963f29bbde36, but since you've already installed (and restarting coreaudiod is the last thing the install script does) you should be able to just do `sudo killall coreaudiod` and then open Background Music.app.

@ariera It might be worth pulling and trying the install script again, just in case, but I think you're running into a different bug. Could you open a separate issue for it and paste in your build_and_install.log?

**Comment by Codematic-AppDev** [comment]
Thanks @kyleneideck. `sudo killall coreaudiod` definitely did the trick. Thanks for the response and keep up the good work.
